sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -801,13 +809,52 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
             webpackResourceFiles,
             entries,
             targetDir,
+            stage.value.data,
             log
           ).to[Set]
         }
 
       cachedActionFunction(monitoredFiles.to[Set]).to[Seq]
     }
 
+  def entryPointTask(
+      stage: TaskKey[Attributed[File]]): Def.Initialize[Task[File]] =
+    Def.taskDyn {
+      val linkerConfig = (scalaJSLinkerConfig in stage).value
+      val linker = (scalaJSLinker in stage).value
+      val linkerTag = (usesScalaJSLinkerTag in stage).value
+      Def
+        .task {
+          val targetDir = (crossTarget in stage).value
+          val logger = streams.value.log
```

Comment:
For better integration with sbt's `last` and `inspect tree`, all accesses to `.value` should be inside `Def.taskDyn { }` but *outside* the inner `Def.task { }`. See also the comment at https://github.com/scala-js/scala-js/blob/41d3e73a5b8920368f489353ee759d6fa5342ca7/sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala#L230-L236